evaluate_library_access_from_output_tsv/create_stratefied_sample_of_dois.R

```diff
@@ -0,0 +1,56 @@
+# Load dependencies ------------------------------------------------------------
+
+# Load magrittr pipe
+`%>%` = dplyr::`%>%`
+
+# Settings ---------------------------------------------------------------------
+
+lzma_compressed_library_access_data_location <- file.path(
+  'data', 'library_coverage_xml_and_fulltext_indicators.tsv.xz'
+)
+
+sample_size_per_cell <- 100  # This will be for each cell, multiplied by 
+# 2 full_text_indicator status
+
+output_tsv_location <- file.path(
+  'evaluate_library_access_from_output_tsv',
+  'manual-doi-checks.tsv'
+)
+
+randomizer_seed_to_set <- 3  # Ensure that random sampling will always return
+# the same result.
+
+# Read the dataset -------------------------------------------------------------
+
+library_access_data <- readr::read_tsv(
+  gzfile(lzma_compressed_library_access_data_location),
```

Comment:
gzfile not needed here. `readr` will detect that path ends in `.xz`. I'm actually surprised gzfile works, given that [wouldn't it be](https://stat.ethz.ch/R-manual/R-devel/library/base/html/connections.html) xzfile?

Reply:
Removed in ccf4a14.

Re: `gzfile`, I didn't actually consider `xzfile` (I didn't know about it, until you mentioned it just now), as `gzfile` was the first thing I found, and it worked. From its manual, 

> For gzfile the description is the path to a file compressed by gzip: it can also open for reading uncompressed files and those compressed by bzip2, xz or lzma.

So, the function's name is possibly confusingly narrow. I wonder whether the `gzfile` R function was developed earlier than `xzfile`?